server/src/game/client_actor.rs

```diff
@@ -111,7 +111,10 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for WsClient {
             ws::Message::Pong(_) => {
                 self.last_heartbeat = Instant::now();
             }
-            ws::Message::Binary(_) => panic!("Unexpected binary"),
+            ws::Message::Binary(_) => {
+                println!("ERROR: Received an unexpected binary");
```

Comment:
We might as well start using the `log` crate now by adding `log = "0.4"` to the dependencies, then just call `log::error!` here.

Reply:
Side note: this should work out of the box since we are already using `env_logger` for actix web server logging.